clients/cloud/ccloud/ccloud-ccsr-example.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+source ../../../utils/helper.sh
+source ../../../utils/ccloud_library.sh 
+
+ccloud::validate_expect_installed \
+  && print_pass "expect installed" \
+  || exit 1
+check_timeout \
+  && print_pass "timeout installed" \
+  || exit 1
+ccloud::validate_version_ccloud_cli 1.13.0 \
+  && print_pass "ccloud version ok" \
+  || exit 1
+ccloud::validate_logged_in_ccloud_cli \
+  && print_pass "logged into ccloud CLI" \
+  || exit 1
+
+if [[ -z "$SR_API_KEY" ]] || [[ -z "$SR_API_SECRET" ]] ; then
+  echo "ERROR: The variables SR_API_KEY and SR_API_SECRET must be set before running this script ('export SR_API_KEY=<>', 'export SR_API_SECRET=<>')"
+  exit 1
+fi
+
+# Set topic name
+topic_name=test2
+
+# Create topic in Confluent Cloud
+echo -e "\n# Create topic $topic_name"
+ccloud kafka topic create $topic_name || true
```

Comment:
Does the `|| true` do something here?

Reply:
No but...this leads to the discovery that the CLI now supports `--if-not-exists` flag ;)

Reply:
💯 💯 💯